src/plays/github-finder/GitHubFinder.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { getPlayById } from "meta/play-meta-util";
+import Input from "./Input";
+import { ButtonSubmit } from "./styles";
+
+import PlayHeader from "common/playlists/PlayHeader";
+
+function GitHubFinder(props: any) {
+  // Do not remove the below lines.
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+  const [username, setUsername] = React.useState("");
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={play} />
+        <div className="play-details-body">
+          {/* Your Code Starts Here */}
+          <div>
+            <h1>GitHub Finder</h1>
+            <p>Your site to find programmers quickly and easily!</p>
+          </div>
+          <iframe
```

Comment:
@Programming-School-Pro-Coding The idea of the app is great but it is not aligned with the phylosphy of ReactPlay because we pull the external app into it. The idea here is for people to learn from the code to get inspired.

May I request you to pull the code into the play than linking using an iframe. In the future we will just link to external app with direct link but not with iframe. Please take ur time and pull the code in.

Thanks,

Reply:
I solve it

Reply:
@Programming-School-Pro-Coding I looked into the code changes. I am sorry if I was not very clear in suggesting the changes last time.

What I meant is, that we want the code for building the play within the ReactPlay code repository. You have a great play for GitHub user finder but the core code(the finding logic, styles, etc) is part of another application and you have just linked it(either anchor tag or href). I'm afraid that, our goal is not that right now. In the future, we will support the external app linking the way you have done.

For now, we want the complete code of the play in the ReactPlay codebase. So for this play you need to bring all the logic code within the ReactPlay codebase(under your play folder). This way, anyone using ReactPlay to learn by reading your code, does not have to toggle between multiple repos. Please make the suggested changes. I think it would be a matter of bringing the component files to your play folder and readjusting a few things. In case, it is not possible now, we can look into this play later when we integrate the external app. Thank you.